src/composables/gmnh-service.ts

```diff
@@ -74,4 +74,38 @@ export async function uploadImage (img: string, walletAddr: PublicKey): Promise<
   };
 
 
+  export async function createGMNHQuestion (img: string, title: string, description: string, walletAddr: PublicKey): Promise<IMintResult> {
+
+    const createQuestionUrl = DEFAULTS.GMNH_SERVICE_APP_URL + 'createQuestion';
+
+    const data = new FormData();
+    data.append('file', img);
+    data.append('title', title);
+    data.append('description', description);
+    data.append('userWalletAddr', walletAddr.toBase58());
+
+    //TODO: ADD ERROR HANDLING
```

Comment:
its a nice to have.  Eventually i need to implement more robust error handling so that when the back-end returns an error, it actually displays that to the user.